src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -83,6 +83,12 @@ function MoneyRequestView({report, parentReport, shouldShowHorizontalRule, polic
         description += ` • ${translate('iou.pending')}`;
     }
 
+    // A temporary solution to hide the transaction detail
```

Comment:
@dukenv0307 What was the plan for this once the transaction is added as a prop (which it is now)?